jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1098,6 +1098,30 @@ def func1(*jax_args):
 
 tf_impl[lax.scan_p] = _scan
 
+def _sort(*operand: tf.Tensor, dimension: int = -1,
+          is_stable: bool = True) -> Tuple[TfVal, ...]:
+  if len(operand) == 0:
+    raise TypeError("Sort requires at least one operand")
+  if len(operand) > 1:
+    raise NotImplementedError("TODO: multiple tensors")
+  dimension = lax._canonicalize_axis(dimension, len(operand[0].shape))
```

Comment:
I believe that we do not need `_canonicalize_axis` because we can assume here that once `lax.sort` has been turned into a `sort_p.bind` all arguments have been sanitized. If you look at `_sort_translation_rule` in `lax.py` you will see that it also does not do any argument sanitization.